### PR TITLE
Updated tests with latest version of @testing-library

### PR DESCRIPTION
**Pull request by robinpiets**
- Updated @testing-library to latest version
- Updated all tests accordingly
- Fixed the broken "should be able to find all Intro pages" test in config.test.js 

---

### CODE REVIEW

**Review by svenjens — APPROVED** [review]

File: apps/client/src/components/Nav.test.js [hunk 1]
```diff
@@ -24,30 +25,36 @@ const Wrapper = ({ children }) => {
...
       <Nav />
     </Wrapper>
   );
 
-  expect(screen.findByText("Vorige")).not.toBeFalsy();
-  expect(screen.findByText("Volgende")).not.toBeFalsy();
+  const prevButton = queryByTestId(PREV_BUTTON);
+  const nextButton = queryByTestId(NEXT_BUTTON);
+
+  expect(nextButton).not.toBeInTheDocument();
```
Comment (afjlambert) [inline apps/client/src/components/Nav.test.js]: if nextButton is only used once I would consider placing it in expect fn directly.
`expect(queryByTestId(NEXT_BUTTON)).not.toBeInTheDocument();`
Minor detail though.

File: apps/client/src/components/Nav.test.js [hunk 2]
```diff
@@ -60,13 +67,13 @@ it("Nav should render with prop values and should fire events", () => {
...
 
-  expect(screen.getByText("Prev")).toBeTruthy();
-  expect(screen.getByText("Next")).toBeTruthy();
-  expect(screen.getByText("Next")).toHaveStyle("margin-right: 10px");
+  expect(getByText("Prev")).toBeInTheDocument();
+  expect(getByText("Next")).toBeInTheDocument();
+  expect(getByText("Next")).toHaveStyle("margin-right: 10px");
 
-  fireEvent.click(screen.getByText("Prev"));
+  fireEvent.click(getByText("Prev"));
```
Comment (afjlambert) [inline apps/client/src/components/Nav.test.js]: Here you actually use previous-button twice, so creating a var could be better :-p

File: apps/client/src/config.test.js [hunk 3]
```diff
@@ -1,48 +1,43 @@
...
+import React from "react";
 import "@testing-library/jest-dom/extend-expect";
 import { render, cleanup } from "./utils/test-utils";
-import { INTRO } from "./utils/test-ids";
-import { topics } from "./config";
+import { topics, isProduction, matomo } from "./config";
+
+afterEach(cleanup);
+
+const tryIntroFile = (intro) => {
```
Comment (afjlambert) [inline apps/client/src/config.test.js:8]: So cool if this works in a stable fashion now. 🙏

File: apps/client/src/config.test.js [hunk 4]
```diff
@@ -1,48 +1,43 @@
...
+  try {
+    require(`./intros/${intro}`);
+    return `Found: ${intro}`;
+  } catch {
+    return `Not found: ${intro}`;
+  }
+};
+
+const AllIntroPages = () =>
```
Comment (afjlambert) [inline apps/client/src/config.test.js:17]: XD

File: apps/client/src/config.test.js [hunk 5]
```diff
@@ -1,48 +1,43 @@
...
+  test("should load correct matomo siteId", () => {
+    expect(matomo.siteId).toBe(isProduction ? 29 : 37);
+  });
+
+  it("should be able to find all Intro pages", () => {
+    const { queryAllByText } = render(<AllIntroPages />);
+    expect(queryAllByText(/Not found/i)).toHaveLength(0);
+  });
+
+  it("should be able to detect a missing Intro page", () => {
```
Comment (afjlambert) [inline apps/client/src/config.test.js:34]: Testing the opposite 🙌 +1

**Review by afjlambert — APPROVED** [review]
Cool to see how you are testing the UI looks a lot like testing the checker component 👍 
Some minor comments inline. Merge at your own will ;-) 

note; did not actually run the tests locally.

File: apps/client/src/components/Nav.test.js (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/components/Nav.test.js]: Great catch! Fixed

File: apps/client/src/components/Nav.test.js (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/components/Nav.test.js]: Great catch! Fixed

**Pull request merged**